api/src/main/java/org/openmrs/module/sync2/api/utils/SyncConfigurationUtils.java

```diff
@@ -29,10 +34,34 @@ public static String readResourceFile(String file) throws SyncException {
         }
     }
 
-    public static SyncConfiguration parseJsonFileToSyncConfiguration(String resourcePath) {
+    public static String readResourceFileAbsolutePath(String file)  {
+        File initialFile = new File(file);
+        InputStream targetStream;
+        try {
+            targetStream = new FileInputStream(initialFile);
+        } catch (FileNotFoundException e) {
+            throw new SyncException("File Not found", e);
+        }
+
+
+        try (InputStream in = targetStream) {
+            if (in == null) {
+                throw new SyncException("Resource '" + file + "' doesn't exist");
+            }
+            return IOUtils.toString(in);
+        } catch (IOException e) {
+            throw new SyncException(e);
+        }
+    }
```

Comment:
Could you simplify it. I think you can remove the first part of the method and modify the second part like this:
`try (InputStream in = new FileInputStream(initialFile)) { `